python/cuml/cuml/internals/array_sparse.py

```diff
@@ -72,7 +72,7 @@ def __init__(
         self,
         data=None,
         convert_to_dtype=False,
-        convert_to_mem_type=None,
+        convert_to_mem_type="device",
```

Comment:
Can you fix-up the class doc-string to actually document this parameter? Right now it is not clear to me whether providing "None", "False" or _falsy_ are all valid inputs.